CMakeLists.txt

```diff
@@ -12,9 +12,20 @@ find_package(Threads REQUIRED)
 
 include(CTest)
 
-set(SRC_LIST "src/DBConnection.cpp" "src/DBConnection.hpp" src/DBConnection2.cpp src/DBConnection2.hpp src/ConnectionDispatcher.cpp src/ConnectionDispatcher.hpp)
-
-add_executable(testrunner "${SRC_LIST}" test/ConnectionTest.cpp test/ConnectionTest2.cpp)
+set(SRC_LIST
+        "src/DBConnection.cpp"
```

Comment:
Strange indent

Reply:
@malirod it's handy to observe code with this indent. I've done this according to the same indent in your `CMakeLists.txt` in `cpputils`

Reply:
Can you please point where you saw similar indent, i cannot find it? Please ensure that you're not using tabs

Reply:
@malirod , oh, sure. Sorry, now I see my fault. Fixed